Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/PatientDiscovery201305Processor.java

```diff
@@ -125,9 +128,9 @@ protected boolean checkPolicy(PRPAIN201306UV02 response, AssertionType assertion
         PatientDiscovery201306PolicyChecker policyChecker = PatientDiscovery201306PolicyChecker.getInstance();
 
         // ************************************************************************************************
+        boolean hasEmptySubject = checkEmptySubject(response);
```

Comment:
CheckEmptySubject will return false if response is empty or has empty subject.  By negating the boolean, we will have the same logic as original one